package.json

```diff
@@ -22,6 +22,7 @@
     "@angular/platform-browser": "~13.0.3",
     "@angular/platform-browser-dynamic": "~13.0.3",
     "@angular/router": "~13.0.3",
+    "prettier": "^2.5.1",
```

Comment:
Add prettier package to start standardising code format to help make code easier to read and also eliminate formatting conflicts